types/index.d.ts

```diff
@@ -0,0 +1,109 @@
+// Minimum TypeScript Version: 3.8
+/// <reference types="node" />
+import type { ConnectionOptions } from 'typeorm';
+import type { IncomingMessage, ServerResponse } from 'http';
+import type { PossibleProviders } from './providers';
+import Adapter from './adapters';
+
+interface InitOptions {
+  site: string;
+  providers: PossibleProviders[];
+  database: ConnectionOptions | string;
```

Comment:
Is this a required option? From the [docs](https://next-auth.js.org/getting-started/example#add-api-route) they say it is optional (only required to persist accounts)

Reply:
Good catch 👏🏻  , @iaincollins could you confirm this? 🙏🏻